tests/test-name-formatting/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub fn format_test_name(name: String) -> String {
+    let name = name.to_lowercase().replace("_", " ");
+    let mut name:Vec<_> = name.split_whitespace().collect();
+    if name[0] == "test"{
```

Comment:
```suggestion
    if name[0] == "test" {
```

I see that the formatter is not automatically run on files in this repository.